src/components/Sum.jsx

```diff
@@ -1 +1,31 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
```

Comment:
please use template literals for such cases
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```